djangocms_version_locking/monkeypatch/admin.py

```diff
@@ -18,12 +18,9 @@ def inner(version, **kwargs):
                 version=version,
                 created_by=version.created_by
             )
-        # A published version has no lock, an existing lock should be removed
-        elif version.state == constants.PUBLISHED:
-            try:
-                VersionLock.objects.get(version=version).delete()
-            except VersionLock.DoesNotExist:
-                pass
+        # A any other state than draft has no lock, an existing lock should be removed
+        elif version.state != constants.PUBLISHED:
```

Comment:
Shouldn't it just be else?

Reply:
You're right, it shouldn't have still be checking for not published either. Means the tests didn't cover this because they should have picked that up. I'll sort that now.